### PR TITLE
FR 721 - Juniper SRX (Branch) SPU CPU, Memory and Flow Session Graphs

### PR DESCRIPTION
**Pull request by centralscrutiniser**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

- Added SPU CPU, Memory and Flow Session Graphs

### CODE REVIEW

File: includes/polling/os/junos.inc.php [hunk 1]
```diff
@@ -1,5 +1,52 @@
 <?php
 
+use LibreNMS\RRD\RrdDefinition;
+
+$oid_list = 'jnxJsSPUMonitoringCurrentFlowSession.0 jnxJsSPUMonitoringCPUUsage.0 jnxJsSPUMonitoringMemoryUsage.0';
```
Comment (laf) [inline includes/polling/os/junos.inc.php]: Can we not get this data into the actual mempool and processors module?

Comment (centralscrutiniser) [reply includes/polling/os/junos.inc.php]: @laf  not sure I understand your request, can you explain?

Comment (laf) [reply includes/polling/os/junos.inc.php]: So we already have discovery and poller modules that deal with memory and processors so adding them here is counter to that + you can't alert on this data. So, if it's possible to get data from snmp that can be used in those modules it should be done there.

**Comment by laf** [comment]
You've removed some config for bluecoat devices here + have removed the cpu and memory stuff.

If it's not possible to get it in the poller modules as I asked earlier then this PR was fine.

**Comment by centralscrutiniser** [comment]
Hi @laf 

The Blue Coat code shouldn't have been in this PR. My apologies. 

The CPU (SPU) is actually already graphed by the existing JunOS processor code. Memory isn't, but as it's a pretty static graph, I've decided to remove it. 

The flow sessions are the interesting part, which is still included. 

Hope that makes sense.

**Comment by laf** [comment]
@centralscrutiniser So you need to re-add the things you've removed and also put your schema changes in to a new file, you can't edit existing ones. Use `252.sql`

**Pull request closed without merging**